packages/terra-modal-manager/src/withModalManager.jsx

```diff
@@ -1,21 +1,16 @@
 import React from 'react';
-import AppDelegate from 'terra-app-delegate';
 /* eslint-disable-next-line import/no-cycle */
 import ModalManager from './ModalManager';
 
 const withModalManager = (Component) => {
   const ModalManagerHOC = ({ app, ...otherProps }) => (
-    <ModalManager app={app}>
+    <ModalManager>
       <Component {...otherProps} />
     </ModalManager>
   );
 
   ModalManagerHOC.WrappedComponent = Component;
 
-  ModalManagerHOC.propTypes = {
```

Comment:
Do you want to take on [this issue](https://github.com/cerner/terra-framework/issues/245) as well while we major version bump this component.

Reply:
Yeah I'm actually going to outright remove withModalManager. It's not necessary, and it will be confusing with the new existence of withDisclosureManager. Consider the issue fixed.